iib/workers/config.py

```diff
@@ -348,6 +363,22 @@ def validate_celery_config(conf: app.utils.Settings, **kwargs) -> None:
             )
 
 
+def _validate_multiple_opm_mapping(iib_ocp_opm_mapping: Dict[str, str]) -> None:
```

Comment:
I would add a check, that the default value is present here.

Reply:
Hmm, in that case I think it will make sense to remove the default value from the mapping if we are setting the default opm in config

Reply:
What about validating that whenever `iib_ocp_opm_mapping` is None the default should be set?

Reply:
Updated the validation such that default to be set, irrespective of iib_ocp_opm_mapping is there or not

Reply:
Thanks for the review :)